symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/config/BdkDatafeedConfig.java

```diff
@@ -70,6 +70,7 @@ public DatafeedAsyncLauncherService datafeedAsyncLauncherService(final DatafeedL
    * Allows to publish application {@link RealTimeEvent} asynchronously from {@link RealTimeEventsDispatcher}.
    */
   @Bean(name = "applicationEventMulticaster")
+  @ConditionalOnProperty(value = "bdk.datafeed.event.async", havingValue = "true", matchIfMissing = true)
```

Comment:
Don't we need to reflect this property in https://github.com/finos/symphony-bdk-java/blob/main/symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/SymphonyBdkCoreProperties.java to make it properly typed ?

Reply:
@symphony-thibault , today's "bdk.datafeed.enabled" property is not reflected to this SymphonyBdkCoreProperties.java either, i was thinking this was on purpose as it is a property used in one place in the autoconfig class only. Let me know if you want to do it.

Reply:
Ok ok @yinan-symphony 👍  Let's keep it as it is for now, especially since the property is now documented